cdap-ui/app/features/tracker/templates/integrations.html

```diff
@@ -0,0 +1,152 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="row">
+  <div class="integration-item">
+    <div class="integration-information">
+      <h3>Cloudera Navigator</h3>
+      <p>Bridging CDAP Metadata with Cloudera’s data management tool</p>
```

Comment:
Why not just change it now in the PR rather than creating a JIRA, prioritizing JIRA and then working on that JIRA ?